TestCase/Managers/InputManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Runtime.Intrinsics.X86;
+
+namespace TestCase.Managers
+{
+    public class InputManager
+    {
+        public int GetInput()
```

Comment:
Could this be a better named method to reflect what it does please. Something like `GetBoardPositions()` or similar.